src/site/content/en/fast/lazy-loading-video/index.md

```diff
@@ -35,15 +35,18 @@ on the `<video>` element may be desirable:
 </video>
 ```
 
+{% Aside 'important' %}
+Video `poster` images may be considered [LCP candidates](/lcp/#what-elements-are-considered). If you decide to lazy load a video's `poster` image, make sure you're not lazy loading them with a JavaScript-based solution (such as [yall.js mentioned below](#libraries)) if they're in the viewport during startup. This is because the [browser preload scanner](/preload-scanner/#lazy-loading-with-javascript) can't discover them as soon as possible, as they will rely on [a data attribute](https://developer.mozilla.org/docs/Learn/HTML/Howto/Use_data_attributes).
```

Comment:
Do you lazy load a video's poster image? Or do you lazy load the video?

```suggestion
Video `poster` images may be considered [LCP candidates](/lcp/#what-elements-are-considered). If you decide to lazy load a video with a `poster` image, make sure you're not lazy loading them with a JavaScript-based solution (such as [yall.js mentioned below](#libraries)) if they're in the viewport during startup. This is because the [browser preload scanner](/preload-scanner/#lazy-loading-with-javascript) can't discover them as soon as possible, as they will rely on [a data attribute](https://developer.mozilla.org/docs/Learn/HTML/Howto/Use_data_attributes).
```